### PR TITLE
Additions to 5x5 scrambles

### PR DESCRIPTION
**Pull request by Rouxles**
This is something that I think should be implemented. This, however, is somewhat more controversial IMO (but I do stand firm that 5BLD must be scrambled correctly)

### CODE REVIEW

**Review by JoaoViniciusSantos — APPROVED** [review]

**Review by andrewtyberg — APPROVED** [review]

File: wca-guidelines.md [hunk 1]
```diff
@@ -94,6 +94,8 @@ To be more informative, each Guideline is classified using one of the following
 - 4d++) [ADDITION] It is permitted for the puzzle to change its orientation when it is moved from the scrambler to the solving station, as long as no one is attempting to influence the randomness of the orientation (see [Regulation A2e1](regulations:regulation:A2e1)).
 - 4f+) [RECOMMENDATION] The WCA Delegate should generate sufficient scramble sequences for the entire competition ahead of time, including spare scramble sequences for extra attempts.
 - 4f++) [REMINDER] If the WCA Delegate generates any additional scramble sequences during the competition, the scramble sequences must be saved.
+- 4g1+) [CLARIFICATION] 5x5x5 Blindfolded scrambles must be scrambled correctly
+- 4g1++) [RECOMMENDATION] A Delegate should only use discretion for allowing incorrect scramble states if it is necessary for the event to be held.
```
Comment (Nevseros) [inline wca-guidelines.md]: Must be `A WCA Delegate`, not just `A Delegate`.

**Review by Nevseros — CHANGES_REQUESTED** [review]

File: wca-guidelines.md (hunk 1 above)
Comment (Rouxles) [reply wca-guidelines.md]:
```suggestion
- 4g1++) [RECOMMENDATION] A WCA Delegate should only use discretion for allowing incorrect scramble states if it is necessary for the event to be held.
```

**Review by CelineTrammi — APPROVED** [review]

**Review by Nevseros — APPROVED** [review]

File: wca-guidelines.md [hunk 2]
```diff
@@ -94,6 +94,8 @@ To be more informative, each Guideline is classified using one of the following
 - 4d++) [ADDITION] It is permitted for the puzzle to change its orientation when it is moved from the scrambler to the solving station, as long as no one is attempting to influence the randomness of the orientation (see [Regulation A2e1](regulations:regulation:A2e1)).
 - 4f+) [RECOMMENDATION] The WCA Delegate should generate sufficient scramble sequences for the entire competition ahead of time, including spare scramble sequences for extra attempts.
 - 4f++) [REMINDER] If the WCA Delegate generates any additional scramble sequences during the competition, the scramble sequences must be saved.
+- 4g1+) [CLARIFICATION] 5x5x5 Blindfolded scrambles must be scrambled correctly
```
Comment (Rouxles) [inline wca-guidelines.md:97]:
```suggestion
- 4g1+) [REMINDER] 5x5x5 Blindfolded scrambles must be scrambled correctly.
```

Comment (Rouxles) [inline wca-guidelines.md:97]:
```suggestion
- 4g1+) [REMINDER] 5x5x5 Blindfolded scrambles must be scrambled correctly
```

Comment (Rouxles) [reply wca-guidelines.md:97]: could also be clarification @lgarron what do you think?

Comment (lgarron) [reply wca-guidelines.md:97]: I think we should still hold off on this change at all until we've figured out a way to address record mis-scrambles (e.g. by requiring scrambler signature and/or an extra checker for it) that can be applied to 5x5x5.

**Comment by Rouxles** [comment]
Will not be implemented for now - is still in consideration for a later update.

**Pull request closed without merging**